src/scanoss/agent/external/src/log.c

```diff
@@ -0,0 +1,101 @@
+

```

Comment:
Can a license header be added to the file?

Reply:
License added

Reply:
Hello @scanoss-qg , unfortunately I can not see any license added to this file on GitHub.